test/shoppingcart_test.rb

```diff
@@ -114,14 +108,14 @@ def test_percentage_occupied
   def test_sorted_products_by_quantity
     cart = ShoppingCart.new("King Soopers", "30items")
     product1 = Product.new(:paper, 'toilet paper', 3.70, '10')
-    product2 = Product.new(:meat, 'chicken', 4.50, '2')
     product3 = Product.new(:paper, 'tissue paper', 1.25, '1')
+    product2 = Product.new(:meat, 'chicken', 4.50, '2')
     product4 = Product.new(:produce, 'apples', 0.99, '20')
 
+    cart.add_product(product4)
```

Comment:
I rearranged these just to make sure that the test still passed if the products were added in a different order!